src/components/WwwFrame/TheHeader.vue

```diff
@@ -476,6 +478,20 @@ import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
 import SearchBar from './SearchBar';
 import PromoCreditBanner from './PromotionalBanner/Banners/PromoCreditBanner';
 
+const hasLentBeforeCookie = 'kvu_lb';
```

Comment:
If you add these to the util file, import and use instead of these two constants.